### ISSUE TITLE
SUMMARY

### ISSUE THREAD

**Issue by sangjunp4rk** [mode-js/Colmar-Academy#5]
Summary Grade: Exceeds Expectations

Good job with your code! I really liked the way you clearly organized your code with whitespace, indentation, and comments. It makes your code really easy to navigate and read, as well as visually pleasing. Also, I really like the way you used classes and ids to group your sections together. As your code grows in size, this will help organize and manage your website. You also gave tremendous effort into your mobile resizing. Great start to the Ready program! 

There really isn’t much to say though since you did very well. You did an exceptional job with your mobile resizing. You website looks good at all sizes! In the future, you can consider helpers and libraries like Bootstrap (http://getbootstrap.com/) to help you build a responsive website. This library does a lot of pre-arranging and adjusting so that you don't have to write such detailed css.

**Comment by js-mode** [comment]
Yes -- right after I finished this project I went to the next chapter of the class and they introduce Bootstrap.  Bootstrap would have made this project a lot easier, but they didn't teach me it until after I was done.  I am glad I was able to finish this project because it makes me appreciate how involved CSS coding can be.  I think bootstrap is a great framework, but for some of the apps I am creating, I don't want to be limited or stuck in that framework and want to put in the extra effort to make sure the CSS is proper.
